suite-native/module-accounts-import/src/screens/XpubScanScreen.tsx

```diff
@@ -216,7 +216,7 @@ export const XpubScanScreen = ({
                         )}
                     </VStack>
                 </Form>
-                {isDevelopOrDebugEnv() && (
+                {!isProduction() && (
```

Comment:
Not sure about this but probably fine

Reply:
I guess I'll change it back if we ever add staging for it to be bullet proof and staging looks like prod

Reply:
https://github.com/trezor/trezor-suite/pull/11720/commits/615a568dc2e045e77767999e7b6c775f44c0dbe5